google-cloud-pubsub/OVERVIEW.md

```diff
@@ -19,96 +19,128 @@ require "google/cloud/pubsub"
 
 pubsub = Google::Cloud::PubSub.new
 
-topic = pubsub.topic "my-topic"
-topic.publish "task completed"
+publisher = pubsub.publisher "my-topic"
+publisher.publish "topic-message"
+
+subscriber = pubsub.subscriber "my-topic-sub"
+subscriber.listen do |received_message|
+  puts "Message: #{received_message.message.data}"
+  received_message.acknowledge!
+end
 ```
 
+This guide provides an overview of the client library's operations, which are categorized into Admin Operations and Data Plane Operations.
+
+* **Admin Operations**: Used for creating, configuring, and managing Pub/Sub resources (topics, subscriptions, schemas).
+* **Data Plane Operations**: For the core functionality of publishing and receiving messages.
+
 To learn more about Pub/Sub, read the [Google Cloud Pub/Sub Overview
 ](https://cloud.google.com/pubsub/overview).
 
-## Retrieving Topics
+## Admin Operations
 
-A Topic is a named resource to which messages are sent by publishers. A Topic is
-found by its name. (See {Google::Cloud::PubSub::Project#topic Project#topic})
+### Topic Admin Client
+
+Manages topic resources.
 
 ```ruby
 require "google/cloud/pubsub"
 
-pubsub = Google::Cloud::PubSub.new
-topic = pubsub.topic "my-topic"
+pubsub = Google::Cloud::PubSub.new project_id: "my-project-id"
+topic_admin = pubsub.topic_admin
 ```
 
-## Creating a Topic
+#### Creating a Topic
 
-A Topic is created from a Project. (See
-{Google::Cloud::PubSub::Project#create_topic Project#create_topic})
+A Topic is a named resource to which messages are sent by publishers. The resource must be created using `TopicAdmin::Client` before it can be used.
 
 ```ruby
-require "google/cloud/pubsub"
+topic_name = "projects/my-project/topics/my-topic"
```

Comment:
In other examples below, you're using `topic_path` and similar path helpers to build this string. But here you're hard-coding the path manually. We should do one or the other consistently in these examples. Either approach is okay; I might lean slightly toward using path helpers, but see my other comment on the visibility of the methods.